Nur in libffado-2.2.1.fixed/admin: dbus.pyc.
Nur in libffado-2.2.1.fixed/admin: doxygen.pyc.
Nur in libffado-2.2.1.fixed/admin: pkgconfig.pyc.
Nur in libffado-2.2.1.fixed/admin: pyuic4.pyc.
Nur in libffado-2.2.1.fixed/admin: pyuic.pyc.
Nur in libffado-2.2.1.fixed/admin: scanreplace.pyc.
Nur in libffado-2.2.1.fixed: cache.
Nur in libffado-2.2.1.fixed: .sconsign.dblite.
Nur in libffado-2.2.1.fixed/support/tools: ffado_diag_helpers.pyc.
diff -ru libffado-2.2.1/tests/test-enhanced-mixer.cpp libffado-2.2.1.fixed/tests/test-enhanced-mixer.cpp
--- libffado-2.2.1/tests/test-enhanced-mixer.cpp	2008-05-29 10:24:21.000000000 +0200
+++ libffado-2.2.1.fixed/tests/test-enhanced-mixer.cpp	2016-02-11 10:12:40.929018609 +0100
@@ -27,6 +27,7 @@
 #include "libieee1394/ieee1394service.h"
 #include <cstdlib>
 #include <cstring>
+#include <cerrno>
 
 using namespace AVC;
 
@@ -77,7 +78,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int node_id = strtol( argv[1], &tail, 0 );
     int fb_id   = strtol( argv[2], &tail, 0 );
diff -ru libffado-2.2.1/tests/test-mixer.cpp libffado-2.2.1.fixed/tests/test-mixer.cpp
--- libffado-2.2.1/tests/test-mixer.cpp	2008-03-23 11:32:19.000000000 +0100
+++ libffado-2.2.1.fixed/tests/test-mixer.cpp	2016-02-11 10:10:46.236452463 +0100
@@ -28,6 +28,7 @@
 
 #include "libieee1394/ieee1394service.h"
 #include <string.h>
+#include <cerrno>
 
 DECLARE_GLOBAL_DEBUG_MODULE;
 
@@ -227,7 +228,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int port = strtol( argv[1], &tail, 0 );
     int node_id = strtol( argv[2], &tail, 0 );
diff -ru libffado-2.2.1/tests/test-pan.cpp libffado-2.2.1.fixed/tests/test-pan.cpp
--- libffado-2.2.1/tests/test-pan.cpp	2008-09-23 23:51:42.000000000 +0200
+++ libffado-2.2.1.fixed/tests/test-pan.cpp	2016-02-11 10:10:07.618935249 +0100
@@ -27,6 +27,8 @@
 
 #include "libieee1394/ieee1394service.h"
 
+#include <cerrno>
+
 const bool bVerbose = false;
 
 using namespace AVC;
@@ -131,7 +133,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int node_id = strtol( argv[1], &tail, 0 );
     int fb_id   = strtol( argv[2], &tail, 0 );
diff -ru libffado-2.2.1/tests/test-volume.cpp libffado-2.2.1.fixed/tests/test-volume.cpp
--- libffado-2.2.1/tests/test-volume.cpp	2008-05-05 23:29:40.000000000 +0200
+++ libffado-2.2.1.fixed/tests/test-volume.cpp	2016-02-11 10:11:13.669109508 +0100
@@ -27,6 +27,8 @@
 
 #include "libieee1394/ieee1394service.h"
 
+#include <cerrno>
+
 const bool bVerbose = false;
 
 using namespace AVC;
@@ -127,7 +129,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int node_id = strtol( argv[1], &tail, 0 );
     int fb_id   = strtol( argv[2], &tail, 0 );
